### PR TITLE
NEXT-26065 - Add scheduler docs

### PR DESCRIPTION
**Pull request by Christian-Rades**

### CODE REVIEW

File: guides/hosting/infrastructure/scheduled-task.md [hunk 1]
```diff
@@ -60,6 +60,25 @@ You can use the flag `--no-wait` and run the command from an operating system sc
...
+This feature is experimental.
+:::
+
+You can run scheduled tasks as part of your queue workers with the help of the symfony scheduler component.
+``` bash
+bin/console messenger:consume scheduler_shopware
+```
+
+On startup of this command reads the `scheduled_task` database table and applies the stored intervals, an entry in this table is optional.
+To deactivate tasks set status to `Shopware\Core\Framework\MessageQueue\ScheduledTask\ScheduledTaskDefinition::STATUS_INACTIVE` in this table, and restart the consume command.
```
Comment (sushmangupta) [inline guides/hosting/infrastructure/scheduled-task.md:79]:
```suggestion
You can run scheduled tasks as part of your queue workers with the help of the symfony scheduler component.

``` bash
bin/console messenger:consume scheduler_shopware
```

On startup of this command reads the `scheduled_task` database table and applies the stored intervals, an entry in this table is optional.
To deactivate tasks, set status to `Shopware\Core\Framework\MessageQueue\ScheduledTask\ScheduledTaskDefinition::STATUS_INACTIVE` in this table, and restart the consume command.
```

**Review by sushmangupta — APPROVED** [review]
Thanks for the update! :)

**Comment by keulinho** [comment]
Hey @sushmangupta , 
this change was just merged into 6.6, should we change the target branch then to 6.6?
Because you merged the main changes into this branch, we can't just change the target branch in the UI :see_no_evil: 

I would also be fine merging it here, as it has a big warning box that it was just added in 6.6

File: guides/hosting/infrastructure/scheduled-task.md [hunk 2]
```diff
@@ -60,6 +60,25 @@ You can use the flag `--no-wait` and run the command from an operating system sc
...
+This feature is experimental.
+:::
+
+You can run scheduled tasks as part of your queue workers with the help of the symfony scheduler component.
+
+``` bash
+bin/console messenger:consume scheduler_shopware
+```
+
+On startup of this command reads the `scheduled_task` database table and applies the stored intervals, an entry in this table is optional.
```
Comment (sushmangupta) [inline guides/hosting/infrastructure/scheduled-task.md:79]:
```suggestion
On startup of this command reads the `scheduled_task` database table and applies the stored intervals, an entry in this table is optional.  In the event that these intervals are modified in the database, it is necessary to restart the command for the updated intervals to take effect.







```

**Comment by sushmangupta** [comment]
To not mess things. I have created a new PR pointing to 6.6 branch : https://github.com/shopware/docs/pull/1253. Hence closing this.

**Pull request closed without merging**